system-int-test/src/cmr/system_int_test/utils/bootstrap_util.clj

```diff
@@ -147,6 +147,18 @@
   ([provider-id headers]
    (bulk-index-by-url (url/bulk-index-subscriptions-url provider-id) headers)))
 
+(defn bulk-index-grids
+  "Call the bootstrap app to bulk index tools (either all of them, or just the
+  ones for the given provider)."
+  ([]
+   (bulk-index-grids {transmit-config/token-header (transmit-config/echo-system-token)} nil nil))
+  ([headers _ _]
```

Comment:
nit: move this to the bottom as it has more parameters than others.

Reply:
I like what others have done where they gave a name to the ignored parameter so that future developers know what is in the argument

Reply:
@jceaser interestingly, in this case it appears those _ parameters are actually always nothing (nil), as a way to reserve the 3-argument case. (bc you can't just pass the URL function an argument if it's nil, as it will return a URL with "/null" at the end) I was following how other concepts had been written. 

However, after some consideration I decided that I agree with you that the blank parameters are more confusing. This question of readability is one of taste. I opted to combine the 2-parameter and 3-parameter versions with an if.